config/k8s/network/ingress.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: ingress
-  namespace: system
+  namespace: flotta
```

Comment:
I can only assume that the same will have to be changed here as well:

https://github.com/project-flotta/flotta-operator/blob/main/config/network/route.yaml#L7

Reply:
done